models/anderson_2007.cpp

```diff
@@ -2,18 +2,27 @@
 // from anderson et al 2007 - Vancomycin pharmacokinetics in preterm neonates and 
 // the prediction of adult clearance.
 [PARAM] @annotated
-CL    : 3.83 : Clearance (L/hr)
-V     : 39.4 : Volume (L) 
-WT    : 2.9  : Weight (kg)
-PMA   : 34.8 : post-menstral age (wk)
-SCR   : 0.9  : serum creatinine (mg/dL)
-VENT  : 0    : Ventillation status
-IONT  : 0    : ionotrope status
-Kage  : 0.00789 : scaling constant of age on renal function
-Fvent : 1.03 : Scaling factor applied for use of positive pressure artificial ventilation
-Fiont : 1.19 : Scaling factor applied for use of inotropes (use of dopamine)
-OCC   : 1     : Occasion
-  
+CL        : 3.83    : Clearance (L/h)
+V         : 39.4    : Volume (L) 
+WT        : 2.9     : Weight (kg)
```

Comment:
The 70 kg weight standard (and male sex and 40 year for age and 6 L/h/70kg for CLcr) are standards that are not necessarily linked to age or species.
Other covariate values e.g. 2.9 kg may be used to illustrate the application of a standard model with a different weight (and weight appropriate covariates such as PMA). But these are not standard values.